apps/nextjs-website/src/components/atoms/CtaSlide/CtaSlide.tsx

```diff
@@ -1,12 +1,14 @@
 'use client';
 import { Box, Stack, Typography, useTheme } from '@mui/material';
 import { ButtonNaked } from '@pagopa/mui-italia';
+import { BlocksContent } from '@strapi/blocks-react-renderer';
 import Image from 'next/image';
 import Link from 'next/link';
 import React, { ReactNode } from 'react';
 
 export type CtaSlideProps = {
   readonly title: string;
+  readonly subhead?: BlocksContent | null;
```

Comment:
Is `null` value a real option? Didn't you used `NullToUndefined` type in `HeroSlideCodec` constant?